modules/@apostrophecms/schema/ui/apos/components/AposInputSlug.vue

```diff
@@ -92,6 +104,7 @@ export default {
               parts.pop();
             }
             parts.push(this.slugify(newValue, { componentOnly: true }));
+            // TODO: handle page archives.
```

Comment:
BTW, I think the deduplication of page slugs is probably a holdover from A2 workflow's "keep the page at the same point in the tree, yet also in the trash! 👻" policy. Their slugs really should just reflect their position in the archive and so a bunch of logic could potentially be removed, with a migration for the existing ones. But, that might be more work than making the current situation work.

Reply:
I added a new ticket for pages. They were not working for different reasons already and this was a chunky one already.